### ISSUE TITLE
feature request: support for TextIterationstreamer of HF

### ISSUE THREAD

**Issue by RachelShalom** [intel/intel-extension-for-transformers#1101]
Hi I want to be able to stream the model not only to stdoutput. the current streamer: TextStreamer only works with the stdouput as I understands it. I tried using the TextIterationStreamer but the current code does not support it

here is a reference:
https://github.com/huggingface/transformers/blob/fc5b7419d4c8121d8f1fa915504bcc353422559e/src/transformers/generation/streamers.py#L125

**Comment by kevinintel** [comment]
Thank to your feedback, we will support it.

**Comment by AdityaKulshrestha** [comment]
@RachelShalom Hi Rachel. I am also trying to demonstrate the inference speed of the llm on intel. Were you able to find any walk around or other method to stream the tokens?

**Comment by RachelShalom** [comment]
@AdityaKulshrestha I  assume there are servings options. @kevinintel did you guys decided to work on it?

**Comment by kevinintel** [comment]
Yes, we will support it recently. After the fature enabled, I will upsate in this issue
